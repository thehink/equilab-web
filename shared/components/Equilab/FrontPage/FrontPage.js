import React, { Component } from 'react';

import Info from 'Equilab/Info';
import Testimonials from 'Equilab/Testimonials';
import AboveFooter from 'Equilab/AboveFooter';

class FrontPage extends Component {
  render() {
    return (
      <div className="front-page">
        {/*  Hero */}
        <Info />
        <Testimonials />
        <AboveFooter />
        FrontPage
      </div>
    );
  }
}

export default FrontPage;
