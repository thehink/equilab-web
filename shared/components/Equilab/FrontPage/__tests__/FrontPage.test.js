/* eslint-disable import/no-extraneous-dependencies */

import React from 'react';
import { shallow } from 'enzyme';

import FrontPage from '../FrontPage';

describe('<FrontPage />', () => {
  test('renders', () => {
    const wrapper = shallow(<FrontPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
