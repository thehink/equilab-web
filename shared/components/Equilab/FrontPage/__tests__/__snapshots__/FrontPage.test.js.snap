// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FrontPage /> renders 1`] = `
<div
  className="front-page"
>
  <Info />
  FrontPage
</div>
`;
